code/modules/reagents/chemistry/machinery/pandemic.dm

```diff
@@ -237,14 +240,18 @@
 		. = TRUE //no afterattack
 		if(machine_stat & (NOPOWER|BROKEN))
 			return
+		var/obj/item/reagent_containers/beaker_out
 		if(beaker)
-			to_chat(user, "<span class='warning'>A container is already loaded into [src]!</span>")
-			return
+			beaker_out = eject_beaker() //now with 100% more swapping
 		if(!user.transferItemToLoc(I, src))
 			return
-
+		if(beaker_out)
+			if(user && Adjacent(user) && user.can_hold_items())
+				user.put_in_hands(beaker_out)
 		beaker = I
 		to_chat(user, "<span class='notice'>You insert [I] into [src].</span>")
+		if(beaker_out) to_chat(user, "<span class='notice'>You remove [beaker_out] and insert [I] into [src].</span>")
+		else to_chat(user, "<span class='notice'>You insert [I] into [src].</span>")
```

Comment:
No control statement may contain code on the same line as the statement, split this into 4 lines

Reply:
oh god

Reply:
I think that I split it up correctly, is it better now?

Reply:
Yep, that's great